draft-lcurley-moq-largest-group.md

```
@@ -0,0 +1,141 @@
+Subscription Filter {
+  Filter Type (vi64) = 0x20
+}
+~~~
+
+The Largest Group filter uses Filter Type value `0x20`.
+No additional fields (Start Location or End Group Delta) are present in the Subscription Filter.
+
+## Semantics
+When the publisher receives a SUBSCRIBE with the Largest Group filter, it computes the start location as:
+
+~~~
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)